src/query/mod.rs

```diff
@@ -15,7 +15,6 @@ pub use combine::{SelectA, SelectB};
 pub use delete::*;
 pub use helper::*;
 pub use insert::*;
-pub use join::*;
```

Comment:
The file contains only impl, so nothing to export

Reply:
In that case, do we need to keep `pub use join::*;`?